pages/1.7/overview/architecture/index.md

```diff
@@ -10,6 +10,7 @@ enterprise: false
 
 <!-- This source repo for this topic is https://github.com/dcos/dcos-docs -->
 
+[![Project Stats](image_url)](link_url)
```

Comment:
same here. Is this by intention? There is a link but don't see where it gets replaced or where `image_url` is coming from